paella-opencast/plugins/es.upv.paella.opencast.loader/05_initdelegate.js

```diff
@@ -43,9 +43,15 @@ function loadOpencastPaellaDCE(containerId) {
 			}
 			paella.load(containerId, {data:data});
 		},
-		function(){
-			// #DCE bypassing paella5 opencast auth, current DCE version is performed within getEpisode()
-			paella.messageBox.showError("Unable to load video " + paella.utils.parameters.get('id'));
+		function(jsonData){
+			// #DCE start specific DCE auth handling, formally in isHarvardDceAuth() (MATT-2212)
+			var dceAuthError = '';
+			if (jsonData && jsonData[ 'dce-auth-results'] && paella.opencast.isHarvardDceAuthRedirect(jsonData, dceAuthError)) {
```

Comment:
If the idea is that `dceAuthError` will get the value assigned in `isHarvardDceAuthRedirect`, it's not going to work. When you declare `dceAuthError` as an arg in `isHarvardDceAuthRedirect` it creates a new variable in that scope. You'll need to pass in an object or something, e.g.:

    var dceAuthError = {};

and then in `isHarvardDceAuthRedirect`...

    dceAuthError['msg'] = " " + authResult.dceErrorMessage;